### PR TITLE
Fix for www bug

### PR DESCRIPTION
**Pull request by jdorfman**

### CODE REVIEW

File: discordbot.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def send_graphql_request(repo_name):
 async def embedding(ctx: discord.ApplicationContext, repo_name: str):
     try:
         await ctx.respond(f"Processing {repo_name}")
-        send_graphql_request(repo_name=repo_name.replace("https://", "").rstrip("/"))
+        send_graphql_request(repo_name=repo_name.replace("https://", "").replace("www", "").rstrip("/"))
```
Comment (andreeleuterio) [inline discordbot.py]: Hey in what cases are we trying to strip www? Here we would be failing processing of repos such as https://github.com/writethedocs/www.

Comment (jdorfman) [reply discordbot.py]: @andreeleuterio good point. This is what I'm trying to fix:

Image 2023-06-07 at 7 34 14 AM

Comment (jdorfman) [reply discordbot.py]: So maybe I do a replace on `"https://www."`?

Comment (andreeleuterio) [reply discordbot.py]: ah okay so I suggest this (after importing the re library):
```
send_graphql_request(repo_name=re.sub(r"^(www\.|https://)(.*?)/?$", r"\2", repo_name)
```

Comment (andreeleuterio) [reply discordbot.py]: This should strip `www.` or `https://` from the beginning of the string and a trailing `/`.

Comment (jdorfman) [reply discordbot.py]: ok I'll do that

Comment (andreeleuterio) [reply discordbot.py]: worth sanity checking me btw 😅

Comment (jdorfman) [reply discordbot.py]: @andreeleuterio https://github.com/sourcegraph/cody-embeddings-discord-bot/pull/6/commits/216a596704d8b90c5fe28e02d7bd9344f06cdbee

**Review by andreeleuterio — APPROVED** [review]

**Pull request merged**